vscode-dotnet-runtime-library/src/Acquisition/GlobalInstallerResolver.ts

```diff
@@ -221,36 +224,39 @@ export class GlobalInstallerResolver {
 
-        releases.forEach(function (release : any)
+        releases.forEach(function (release: any)
         {
             // eslint-disable-next-line prefer-spread
             sdks.push.apply(sdks, release[releasesKeyAlias]);
         });
 
         for (const sdk of sdks)
         {
-            const thisSDKVersion : string = sdk[this.releasesSdkVersionKey];
-            if(thisSDKVersion === specificVersion) // NOTE that this will not catch things like -preview or build number suffixed versions.
+            const thisSDKVersion: string = sdk[this.releasesSdkVersionKey];
+            if (thisSDKVersion === specificVersion) // NOTE that this will not catch things like -preview or build number suffixed versions.
             {
-               const thisSDKFiles = sdk[this.releasesSdkFileKey];
-               for (const installer of thisSDKFiles)
-               {
-                    if(installer[this.releasesSdkRidKey] === desiredRidPackage && this.installerMatchesDesiredFileExtension(specificVersion, installer, convertedOs))
+                const thisSDKFiles = sdk[this.releasesSdkFileKey];
+                for (const installer of thisSDKFiles)
+                {
+                    if (installer[this.releasesSdkRidKey] === desiredRidPackage && this.installerMatchesDesiredFileExtension(specificVersion, installer, convertedOs))
                     {
                         const installerUrl = installer[this.releasesUrlKey];
-                        if(installerUrl === undefined)
+                        if (installerUrl === undefined)
                         {
                             const releaseJsonErr = new DotnetInvalidReleasesJSONError(new EventBasedError('DotnetInvalidReleasesJSONError',
-                            `URL for ${desiredRidPackage} on ${specificVersion} is unavailable:
+                                `URL for ${desiredRidPackage} on ${specificVersion} is unavailable:
 The version may be Out of Support, or the releases json format used by ${indexUrl} may be invalid and the extension needs to be updated.`),
                                 getInstallFromContext(this.context));
                             this.context.eventStream.post(releaseJsonErr);
                             throw releaseJsonErr.error;
                         }
-                        if(!(installerUrl as string).startsWith('https://download.visualstudio.microsoft.com/'))
+                        if (!this.startsWithAny((installerUrl as string), [
+                            'https://download.visualstudio.microsoft.com/', 'https://builds.dotnet.microsoft.com/', 'https://dotnetcli.trafficmanager.net/',
+                            'https://builds.dotnet.microsoft.com.edgesuite.net/', 'https://dotnetcli.azureedge.net/', 'https://dotnetcli.blob.core.windows.net/',
+                        ]))
```

Comment:
```suggestion
                            'https://download.visualstudio.microsoft.com/', 'https://builds.dotnet.microsoft.com/', 'https://ci.dot.net',
                       'https://dotnetcli.blob.core.windows.net/',
                        ]))
```

These are the only URLs that I can imagine ever using. We don't want the other URLs being documented via code and never plan to use them directly.